python/docs/source/user_guide/dataframe_creation.rst

```diff
@@ -0,0 +1,179 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+==================
+DataFrame Creation
+==================
+
+.. currentmodule:: pyspark.sql
+
+Basic Data Structures
+---------------------
+
+PySpark provides an important class for handling data:
+
+1. :class:`DataFrame`: a distributed collection of data grouped into named columns.
+
+Creating Through `createDataFrame`
+----------------------------------
+
+A PySpark :class:`DataFrame` can be created via :meth:`SparkSession.createDataFrame` typically by passing
+a list of lists, tuples, dictionaries and :class:`Row`, a pandas :class:`pandas.DataFrame`,
+a NumPy :class:`numpy.ndarray` and an :class:`pyspark.RDD`.
+:meth:`SparkSession.createDataFrame` takes the `schema` argument to specify the schema of the :class:`DataFrame`.
+When it is omitted, PySpark infers the corresponding schema by taking a sample from the data.
+
+Creating a PySpark :class:`DataFrame` from a list of lists
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    df = spark.createDataFrame([['Alice', 1], ['Bob', 5]])
+    df
```

Comment:
what about using `df.show()` in the examples instead to show the data?

Reply:
Okay